src/App.js

```diff
@@ -1,24 +1,33 @@
 import React from 'react';
 import './App.css';
-
 import todos from './api/todos';
 import users from './api/users';
+import './style.css';
+import TodoList from './components/todoList/todoList';
+
+function getTodoWithUsers(todo, user) {
+  return todo.map(todoItem => (
+    {
+      ...todoItem,
+      user: user
+        .filter(userItem => userItem.id === todo.userId),
```

Comment:
Такой пользователь будет только один
```suggestion
        .find(userItem => userItem.id === todo.userId),
```